CMakeLists.txt

```
@@ -15,6 +15,7 @@ include_directories(
 
 if(WIN32)
```

```suggestion
if(MSVC)
```
To take into account also UWP

---

Thanks added and working!